defaults/config.js

```diff
@@ -297,25 +297,25 @@ module.exports = {
 	//   For example:
 	//
 	//   ```json
-	//   {
+	//   webirc: {
 	//     "irc.example.net": "password1",
-	//     "irc.example.org": "passw0rd"
-	//   }
+	//     "irc.example.org": "passw0rd",
+	//   },
```

Comment:
I'd either remove this one, or add one at L317 as well, for consistency.

Reply:
Add a comma.